server/Context/OfficeDbContextLokal.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.EntityFrameworkCore;
+using server.Data;
+using server.Models.Domain;
+
+namespace server.Context;
+
+    public class OfficeDbContextLokal : DbContext
```

Comment:
Lokal => Local ?

Reply:
typo, for fort gjord feil med norsk/english